app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala

```diff
@@ -51,21 +61,26 @@ case class CheckYourAnswersViewModel(paymentFrequency: String,
       controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.payslipDeductionsPage().url
     )
 
+    val taxablePayMessages = Map(MONTHLY -> "tai.estimatedPay.update.checkYourAnswers.taxablePay.month",
+                                 WEEKLY -> "tai.estimatedPay.update.checkYourAnswers.taxablePay.week",
+                                 FORTNIGHTLY -> "tai.estimatedPay.update.checkYourAnswers.taxablePay.2week",
+                                 OTHER -> "tai.estimatedPay.update.checkYourAnswers.taxablePay.days")
+
     val taxablePayConfirmationLine = createCheckYourAnswerConfirmationLine(
-      messages("tai.estimatedPay.update.checkYourAnswers.taxablePay", timePeriod(paymentFrequency)),
+      messages(dynamicTitle(Some(paymentFrequency), payPeriodInDays, taxablePayMessages)),
       taxablePay,
       controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.taxablePayslipAmountPage().url,
       isMonetaryValue
     )
 
     val hasBonusOrOvertimeConfirmationLine = createCheckYourAnswerConfirmationLine(
-      messages("tai.estimatedPay.update.checkYourAnswers.hasBonusOrOvertime",currentTaxYearRangeHtmlNonBreakBetween),
+      messages("tai.estimatedPay.update.checkYourAnswers.hasBonusOrOvertime",TaxYearRangeUtil.currentTaxYearRangeBetweenDelimited),
       Some(messages(s"tai.label.${hasBonusOrOvertime.toLowerCase}")),
```

Comment:
Think neda is wanting these to be currentTaxYearRangeSingleLine, we should probably update the deprecated line to reflect this

Reply:
I believe it is just headings